scripts/zones/Chocobo_Circuit/npcs/Adrian.lua

```diff
@@ -1,7 +1,8 @@
 -----------------------------------
 -- Area: Chocobo_Circuit
---  NPC: Adrian
--- Standard Info NPC
+-- NPC: Adrian
+-- Abyssea Support NPC
```

Comment:
I assumed this was a copy/paste error, but no, there's Abyssea NPCs in here!

https://ffxiclopedia.fandom.com/wiki/Adrian

Reply:
yup, same page I got the info from